dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/ESResponseException.java

```diff
@@ -0,0 +1,16 @@
+package com.dotcms.content.elasticsearch.business;
+
+import org.elasticsearch.action.support.master.AcknowledgedResponse;
+
+public class ESResponseException extends Exception {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Classes implementing Serializable should set a serialVersionUID](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5642172)